libs/shared/common/src/lib/components/avatar/avatar.component.ts

```diff
@@ -0,0 +1,32 @@
+import { CommonModule } from '@angular/common';
+import { Component, Input, signal } from '@angular/core';
+import { User } from 'firebase/auth';
+
+import { ExtractInitialsPipe } from '../../pipes';
+import { TooltipModule } from '../tooltip';
+
+@Component({
+  selector: 'expenses-tracker-avatar',
+  standalone: true,
+  imports: [CommonModule, TooltipModule, ExtractInitialsPipe],
+  template: `<div
+      class="p-4 border rounded-full cursor-pointer w-14 h-14 bg-color-avatar-bg border-color-border-default text-color-fg-default"
+      [expensesTrackerTooltip]="ownerTooltip">

```

Comment:
Rename to `userTooltip`

Reply:
Resolved in https://github.com/usamazansari/expenses-tracker/pull/70/commits/f794d33eb05b30d060e8522aaf26af0911802c5d